docs/references/conventions.rst

```diff
@@ -1,46 +1,34 @@
 .. _conventions:
 
-Front-end code conventions
-==========================
+Frontend code conventions
+=========================
 
+In general, follow the `style guide <http://kolibridemo.learningequality.org/style_guide>`_ and use built-in components and conventions when ever possible.
```

Comment:
when ever -> whenever

Reply:
remove the underscores before 'and use'

Reply:
> remove the underscores before 'and use'

that's part of reStructuredText's horrible horrible syntax

Reply:
(even the name reStructuredText is horrible)